app/models/concerns/foreman_remote_execution/host_extensions.rb

```diff
@@ -54,6 +54,17 @@ def execution_interface
       get_interface_by_flag(:execution)
     end
 
+    def set_execution_interface(identifier)
+      if interfaces.find_by(identifier: identifier).nil?
+        msg = (N_("Host's interface '%s' not found.") % identifier)
```

Comment:
how about "Interface with the '%s' identifier was specified as a remote execution interface, however the interface was not found on the host. If the interface exists, it needs to be created in Foreman during the registration."